grouper/fe/handlers/user_disable.py

```diff
@@ -25,7 +25,11 @@ def post(self, user_id=None, name=None):
         if not self.check_access(self.session, self.current_user, user):
             return self.forbidden()
 
-        disable_user(self.session, user)
+        if user.role_user:
+            disable_service_account(self.session, user=user)
```

Comment:
it'd be great if the API for `disable_service_account` was the same as that for `disable_user`